sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
+       =======
+       Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Value
+              The value around which ``x`` is calculated. Can be ``oo`` or
+              ``-oo``.
+
+       n    : Value
+              The number of terms upto which the series is to be expanded.
+
+       dir  : String
+              Optional (default: "+")
+              The series-expansion can be bi-directional. If ``dir="+"``,
+              then (x->x0+). If ``dir="-", then (x->x0-). For infinite
+              ``x0`` (``oo`` or ``-oo``), the ``dir`` argument is determined
+              from the direction of the infinity (i.e., ``dir="-"`` for
+              ``oo``).
+       Examples
+       ========
+
+       >>> from sympy import Symbol, series, tan, oo
+       >>> from sympy.abc import x
+       >>> f = tan(x)
+       >>> f.series(x, 2, 6, "+")
```

Comment:
It looks like it calls `series(self, x=None, x0=0, n=6, dir="+", logx=None)` 
instead of `series(expr, x=None, x0=0, n=6, dir="+")`
which is `expr.series()`

Similarly, the next examples.

Reply:
``f `` is an expression and ``f.series()`` is similar to ``expr.series()``

Reply:
But isn't this function supposed to something like `series(f, x, 2, 6, "+")` ?

Reply:
I have changed it.